examples/add_points_with_features.py

```diff
@@ -29,22 +29,22 @@
 face_color_cycle = ['blue', 'green']
 
 # create a points layer where the face_color is set by the good_point feature
-# and the edge_color is set via a color map (grayscale) on the confidence
+# and the border_color is set via a color map (grayscale) on the confidence
 # feature.
 points_layer = viewer.add_points(
     points,
     features=features,
     size=20,
-    edge_width=7,
-    edge_width_is_relative=False,
-    edge_color='confidence',
-    edge_colormap='gray',
+    border_width=7,
+    border_width_is_relative=False,
+    border_color='confidence',
+    border_colormap='gray',
     face_color='good_point',
     face_color_cycle=face_color_cycle
 )
 
```

Comment:
I do not find deprecation (argument rename) for `add_points`.

Reply:
sorry @Czaki , I don't understand your comment, could you elaborate a bit more?

Reply:
@JoOkuma You should decorate the points class with the `rename_argument` decorator indicating that `edge_width` has been named to `border_width`, and so on.

https://github.com/napari/napari/blob/e2371d708a50bf0090b6feed3f00117f7c1b8182/napari/utils/migrations.py#L7

Do a `git grep rename_argument` to find example usages. The version numbers should be 0.6.0 since 0.5.0

Reply:
and I'm assuming the `add_<layer>` and `view_<layer>` methods don't need to have the deprecation warning because they'll be triggered when creating the `Points` class.